tssc/step_implementer.py

```diff
@@ -224,8 +224,9 @@ def _validate_runtime_step_config(self, runtime_step_config):
         """
         missing_required_config_keys = []
         for required_config_key in self.required_runtime_step_config_keys():
-            if (required_config_key not in runtime_step_config) or \
-                    (not runtime_step_config[required_config_key]):
+            if ((required_config_key not in runtime_step_config) or \
+                    (not runtime_step_config[required_config_key]) and  \
+                    (not isinstance(runtime_step_config[required_config_key], bool))):
```

Comment:
Added in separate branch (NAPSSPO-794), but also here to allow all tests to pass successfully.